test/end-to-end/pages/IDAM/signIn.js

```diff
@@ -21,7 +23,7 @@ module.exports = async function (noScreenerQuestions = false) {
         await I.waitForText('Sign in', testConfig.TestWaitForTextToAppear, 'h1');
         await I.fillField('username', process.env.testCitizenEmail);
         await I.fillField('password', process.env.testCitizenPassword);
-
-        await I.navByClick({css: 'input.button[value="Sign in"]'});
+        await I.navByClick(contentEn.signIn);
+        await I.wait(10);
```

Comment:
please remove this 10 second wait and replace with an implicit wait - possibly changes in navbyclick have caused this? 10 second wait not helpful for devs who may have to continually re-run scripts whilst debugging or adding new test code, thanks